dashboard/app/controllers/registrations_controller.rb

```diff
@@ -59,6 +61,9 @@ def account_type
   #
   def login_type
     @is_signed_out = current_user.nil?
+    location = Geocoder.search(request.ip).try(:first)
+    country_code = location&.country_code.to_s.upcase
+    @in_strict_password_country = Services::User::PasswordChecker.strict_password_country?(country_code)
```

Comment:
On the `/users/sign_up/account_type` page, where the user selects either the Student or Teacher sign-up flow, we need to pass the selected `user_type` as a parameter to the next  (`/users/sign_up/login_type`) page:
```suggestion
    location = Geocoder.search(request.ip).try(:first)
    @country_code = location&.country_code.to_s.upcase
    @user_type = params[:user_type]
```